simplesource-command-serialization/src/main/java/io/simplesource/kafka/serialization/json/JsonAggregateSerdes.java

```diff
@@ -272,7 +273,8 @@ public JsonElement serialize(
         private JsonElement serializeReason(final CommandError commandError) {
             final JsonObject wrapper = new JsonObject();
             wrapper.addProperty(ERROR_MESSAGE, commandError.getMessage());
-            wrapper.addProperty(ERROR_CODE, commandError.getReason().name());
+            wrapper.addProperty(ERROR_CLASS, commandError.getClass().getCanonicalName());
```

Comment:
`ERROR_MESSAGE ` and `ERROR_CLASS` are here for debug purpose. They're not used in the deserialization mechanism.